src/routes/transactions/entities/transaction-info.entity.ts

```diff
@@ -1,12 +1,16 @@
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { RichHumanDescriptionFragment } from '@/routes/transactions/entities/human-description.entity';
 
 export class TransactionInfo {
   @ApiProperty()
   type: string;
   @ApiPropertyOptional({ type: String, nullable: true })
-  humanDescription: string | null;
+  humanDescription: RichHumanDescriptionFragment[] | null;
```

Comment:
⚠️ This is API breaking – merging this would break clients that are currently expecting a `string`.

Reply:
How do we deal with that? Practically, this should only be in use by the web client so far but there is no explicit dependency so increasing the version number by a breaking change wouldn't help. On the other hand adding the new change in a backwards compatible way seems excessive.

Reply:
When working with the API which is exposed to the any client we should always consider on how the contract between the clients and the service breaks.

That's one of the reasons why I've also suggested to use a feature flag so that the API can evolve without breaking the client-side.

This should always be considered when working with an API contract as we might have 3rd parties to the service that expect `X` but get `Y` after some time.

What happens if this gets merged as-is on the current state of the Safe Web App?

Reply:
We have discussed this internally and the feedback was already addressed in the PR.